sunpy/net/dataretriever/sources/tests/test_adapt.py

```diff
@@ -0,0 +1,93 @@
+import pytest
```

Comment:
I would actually suggest for the tests is to copy the tests from another client and just change the imports to the adapt client and any adapt specific attrs or values.

It will avoid you having to write the tests from scratch and it will cover the main areas we need to test for.